### DISCUSSION TITLE
didn't get education benefits aafter 3 days even id gets approved

### DISCUSSION THREAD

**Discussion by ayushbansal289** [community/community#129490] [GitHub Education] [answered]
### Hi everyone,

i didn't get any education benefits even after my id gets approved already 3 days passed

**Comment by davevad93** [comment, marked as answer]
Hi @ayushbansal289 ,

It may take up to a couple of weeks for the benefits to become available, it usually takes just a few days but the waiting time may vary depending on different factors such as the volume of applications. 

When the benefits are available the status will change to:

`Congrats! Your request for @ayushbansal289 was approved and your benefits are now available`

**Reply by rezaikbal** [reply]
How long should I wait for the status of "Congrats! Your request for @myusername was approved and your benefits will be available soon." I became fully benefited?

**Reply by davevad93** [reply]
Hey @rezaikbal  , take a look at this:

![46c34](https://github.com/community/community/assets/94308234/fea13103-6c6d-43e3-a4d7-a399c5210dd1)

**Reply by rezaikbal** [reply]
@davevad93 I submit using my campus email. If my status is green like the screenshot below, is it likely that the Approved status will change to purple? Or could it be rejected?
![Screenshot (2840)](https://github.com/community/community/assets/59811589/a014e079-e667-4937-afb7-5552eb5e3dff)

**Reply by davevad93** [reply]
@rezaikbal , don't worry you have been approved now you just have to wait for the benefits to be available! 👍

**Reply by rezaikbal** [reply]
@davevad93 Thank you very much for the info. And is this GitHub Dev Student Pack only for 1 year or can I still use it as long as I am an active student?

**Reply by davevad93** [reply]
You're welcome @rezaikbal , student validation lasts for 2 years, after which you will need to revalidate it by reapplying again. If you will still be an active student you will be approved.

**Reply by rezaikbal** [reply]
@davevad93 Oh I see, okay, thank you very much for all the information you provided, it really answered all my questions😊

**Reply by devvspaces** [reply]
Mine is violet and not yet gotten benefits

**Reply by davevad93** [reply]
Hey @devvspaces , try first going to the [pack offers page](https://education.github.com/pack) and select one of the available offers, if you can access them you are fine, if not I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

**Comment by kanduru-abhiram** [comment]
Hi @ayushbansal289, please wait till the background color of Approved turns to Violet, once turned, you'll get the access to all the benefits.

![image](https://github.com/community/community/assets/85011280/69f6be71-f32c-4d73-a754-1a89b1fcabf2)
